src/Service.Host/client/src/components/stockLocatorUtility/StockViewerOptions.js

```diff
@@ -31,6 +37,21 @@ function StockViewerOptions({
     const [batcheRef, setBatchRef] = useState('');
     const [inspectedState, setInspectedState] = useState('');
 
+    const table = {
+        totalItemCount: stockLocatorBatches.length,
+        rows: stockLocatorBatches?.map((item, i) => ({
+            id: `${item.id}`,
+            values: [
+                { id: `${i}-0`, value: `${item.batchRef}` },
+                { id: `${i}-1`, value: `${item.partNumber || ''}` },
+                { id: `${i}-1`, value: `${item.palletNumber || ''}` },
+                { id: `${i}-1`, value: `${item.storagePlaceName || ''}` },
+                { id: `${i}-1`, value: `${item.stockRotationDate.split('T')[0] || ''}` }
```

Comment:
Should these number 1s meant to be incrementing?

Reply:
yes